lib/utilities/paginator.rb

```diff
@@ -0,0 +1,85 @@
+module Utilities
+  module Paginator
+    def items_to_display(item:, items_per_page:)
+      @items_per_page = items_per_page || 5
+      @current_page = params[:page] ? params[:page].to_i : 1
+      @number_of_items = item.all.count
```

Comment:
Active record has a method to get the number of DB records for a class. The only reason to perform this computation in ruby would be for performance (but you have not implemented the rest of the changes needed to actually optimise for this)